wolfcrypt/src/ecc.c

```diff
@@ -5564,12 +5564,12 @@ int wc_ecc_sign_hash_ex(const byte* in, word32 inlen, WC_RNG* rng,
                    if (err != MP_OKAY) break;
 
                    /* e = e/k.b */
-                   err = mp_mulmod(&pubkey->k, e, curve->order, e);
+                   err = mp_mulmod(&pubkey->k, e, curve->order, &pubkey->k);
```

Comment:
Please change the comments too, the result is no longer e.

Reply:
Fixed. Made comments and code more inline (added temps with useful names).